man/tpm2_hierarchycontrol.1.md

```diff
@@ -0,0 +1,77 @@
+% tpm2_hierarchycontrol(1) tpm2-tools | General Commands Manual
+%
+% July 2019
+
+# NAME
+
+**tpm2_hierarchycontrol**(1) - Enable and disable use of a hierarchy and its
+associated NV storage.
+
+# SYNOPSIS
+
+**tpm2_hierarchycontrol** [*OPTIONS*] _OPERATION_
+
+# DESCRIPTION
+
+**tpm2_hierarchycontrol**(1) - Allows user change phEnable, phEnableNV,
+shEnable and ehEnable when the proper authorization is provided. Authorization
+ should be one out of owner hierarchy auth, endorsement hierarchy auth and
+ platform hierarchy auth. **As an argument the tool takes the \_OPERATION\_ as
+ string clear|set to clear or set the TPMA_STARTUP_CLEAR bits.**
+Note: If password option is missing, assume NULL.
+
+# OPTIONS
+
+  * **-C**, **\--context-parent**=_PARENT\_CONTEXT\_OBJECT_:
+
+    Context object loaded object's parent. Either a file or a handle number.
+    See section "Context Object Format".
+
+  * **-P**, **\--auth-parent**=_PARENT\_KEY\_AUTH_:
+
+    Optional authorization value to use the parent object specified by **-C**.
+    Authorization values should follow the "authorization formatting standards",
+    see section "Authorization Formatting".
```

Comment:
There is no parent object for this command, so it's just -c and -p and its the hierarchy auth itself

Reply:
okay, could you suggest the short option for 'property'?

Reply:
Options need to change:
  - -p should be -P and --auth-key should be --auth-hierarchy.
  - -c should be -C and --context-object should be --hiearchy.

Reply:
> Options need to change:
> 
> * -p should be -P and --auth-key should be --auth-hierarchy.
> * -c should be -C and --context-object should be --hiearchy.

Okay, I will commit again with this change.